assembly/distributed/bindings.ts

```diff
@@ -29,7 +29,7 @@ export namespace distributed {
    */
   // @ts-ignore: decorator
   @external("lunatic::distributed", "send")
-  export declare function send(node_id: u64, process_id: u64): void;
+  export declare function send(node_id: u64, process_id: u64): ErrCode;
```

Comment:
CI doesn't like this change, because the Lunatic version that changes this API hasn't been released yet.